### PR TITLE
Package into correct dirs

### PR DESCRIPTION
**Pull request by Wovchena**

### CODE REVIEW

**Comment by Wovchena** [comment]
@apaniukov, @mryzhov, @mryzhov

**Comment by apaniukov** [comment]
We don't need to apply this folder structure when building the wheel, that is why the tests failed.

File: src/CMakeLists.txt
Comment (Wovchena) [inline src/CMakeLists.txt:1]: > We don't need to apply this folder structure when building the wheel, that is why the tests failed.

Fixed. The tests still fail but they fail in empty PR the same way: https://github.com/openvinotoolkit/openvino_tokenizers/pull/149

**Review by apaniukov — APPROVED** [review]

**Comment by apaniukov** [comment]
build_jenkins

**Comment by akladiev** [comment]
build_jenkins

**Comment by akladiev** [comment]
build_jenkins

**Pull request merged**